lxd/instance/instance_utils.go

```diff
@@ -678,7 +678,7 @@ func ValidName(instanceName string, isSnapshot bool) error {
 
 		// Snapshot part is more flexible, but doesn't allow space or / character.
 		if strings.ContainsAny(snapshotName, " /") {
-			return fmt.Errorf("Invalid instance snapshot name: Cannot contain space or / characters")
+			return fmt.Errorf("Invalid instance snapshot name: Cannot contain spaces or slashes")
```

Comment:
In commit https://github.com/canonical/lxd/pull/13261/commits/1009cf770710c4ba3c9f437d1daaf4388c9e80b3 you are using `%q` to also indicate the name of the snapshot. Would this be applicable here too?

Reply:
yeah i think it would

Reply:
Fixed in https://github.com/canonical/lxd/pull/13261/commits/1009cf770710c4ba3c9f437d1daaf4388c9e80b3

Reply:
Oh sorry i misunderstood your point

Reply:
fixed